docs/code_navigation/how-to/index_other_languages.mdx

```diff
@@ -4,48 +4,43 @@ This guide is meant to provide instructions for how to enable precise code navig
 The general steps for enabling precise code navigation are as follows:
 
 1. Install Sourcegraph CLI
-2. Install LSIF Indexer
-3. Generate LSIF data
-4. Upload LSIF data
+2. Install SCIP Indexer
+3. Generate SCIP data
+4. Upload SCIP data
```

Comment:
I think by fixing the heading structure in this doc, we should be able to get the automatic ToC generation to show the level 3 headings on the ToC on the right side.

https://sourcegraph.com/docs/code_navigation/how-to/index_other_languages

That should remove the need for this list.